[BUG] [netty-websocket/ERROR]: Fatal error java.net.BindException: Address already in use: bind

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.27.0-0.27.1?

### Operating System

Windows 10

### Minecraft Version

1.21.4

### Describe the bug

minecraft open up download some stuff fine but the ui not are there cant do anything and theres the error window pops up 

sorry if this is dumb

### Steps to reproduce

launch liquidbounce v0.27.0-0.27.1?

### Client Log

```shell
[20:06:20] [main/INFO]: Loading Minecraft 1.21.4 with Fabric Loader 0.16.10
[20:06:20] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
[20:06:20] [ForkJoinPool-1-worker-2/WARN]: Mod com_github_oryxel_cubeconverter uses the version 25bc6ce1b1 which isn't compatible with Loader's extended semantic version format (Could not parse version number component '25bc6ce1b1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[20:06:20] [ForkJoinPool-1-worker-4/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR3-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR3'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[20:06:21] [main/INFO]: Loading 134 mods:
	- fabric-api 0.118.0+1.21.4
	   |-- fabric-api-base 0.4.54+b47eab6b04
	   |-- fabric-api-lookup-api-v1 1.6.86+b1caf1e904
	   |-- fabric-biome-api-v1 15.0.6+b1c29d8e04
	   |-- fabric-block-api-v1 1.0.31+7feeb73304
	   |-- fabric-block-view-api-v2 1.0.19+7feeb73304
	   |-- fabric-blockrenderlayer-v1 2.0.8+7feeb73304
	   |-- fabric-client-tags-api-v1 1.1.29+20ea1e2304
	   |-- fabric-command-api-v1 1.2.62+f71b366f04
	   |-- fabric-command-api-v2 2.2.41+e496eb1504
	   |-- fabric-commands-v0 0.2.79+df3654b304
	   |-- fabric-content-registries-v0 9.1.19+25d1a67604
	   |-- fabric-convention-tags-v1 2.1.17+7f945d5b04
	   |-- fabric-convention-tags-v2 2.13.0+360374ac04
	   |-- fabric-crash-report-info-v1 0.3.6+7feeb73304
	   |-- fabric-data-attachment-api-v1 1.6.0+4e7c604904
	   |-- fabric-data-generation-api-v1 22.2.12+60b6f1b704
	   |-- fabric-dimensions-v1 4.0.10+7feeb73304
	   |-- fabric-entity-events-v1 2.0.14+da99546b04
	   |-- fabric-events-interaction-v0 4.0.3+a4eebcf004
	   |-- fabric-game-rule-api-v1 1.0.63+7d48d43904
	   |-- fabric-item-api-v1 11.3.0+ee91fa1f04
	   |-- fabric-item-group-api-v1 4.2.2+fcb9601404
	   |-- fabric-key-binding-api-v1 1.0.57+7d48d43904
	   |-- fabric-keybindings-v0 0.2.55+df3654b304
	   |-- fabric-lifecycle-events-v1 2.5.4+bf2a60eb04
	   |-- fabric-loot-api-v2 3.0.38+3f89f5a504
	   |-- fabric-loot-api-v3 1.0.26+203e6b2304
	   |-- fabric-message-api-v1 6.0.25+7feeb73304
	   |-- fabric-model-loading-api-v1 4.3.0+ae23723504
	   |-- fabric-networking-api-v1 4.3.10+8998135104
	   |-- fabric-object-builder-api-v1 18.0.13+7d48d43904
	   |-- fabric-particles-v1 4.0.14+7feeb73304
	   |-- fabric-recipe-api-v1 8.1.0+640e77ae04
	   |-- fabric-registry-sync-v0 6.1.9+360374ac04
	   |-- fabric-renderer-api-v1 5.0.3+50f0feb204
	   |-- fabric-renderer-indigo 2.0.3+50f0feb204
	   |-- fabric-rendering-data-attachment-v1 0.3.57+73761d2e04
	   |-- fabric-rendering-fluids-v1 3.1.19+7feeb73304
	   |-- fabric-rendering-v1 10.2.1+0d31b09f04
	   |-- fabric-resource-conditions-api-v1 5.0.13+203e6b2304
	   |-- fabric-resource-loader-v0 3.1.1+360374ac04
	   |-- fabric-screen-api-v1 2.0.38+7feeb73304
	   |-- fabric-screen-handler-api-v1 1.3.116+7feeb73304
	   |-- fabric-sound-api-v1 1.0.32+7feeb73304
	   |-- fabric-tag-api-v1 1.0.7+7d48d43904
	   |-- fabric-transfer-api-v1 5.4.9+efa825c904
	   \-- fabric-transitive-access-wideners-v1 6.3.2+56e78b9b04
	- fabric-language-kotlin 1.13.1+kotlin.2.1.10
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.1.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.1.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.1.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.1.10
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.27.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.10.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.10.1
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.6.1
	   |-- org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm 0.6.0
	   |-- org_jetbrains_kotlinx_kotlinx-io-core-jvm 0.6.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.8.0
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.8.0
	- fabricloader 0.16.10
	   \-- mixinextras 0.4.1
	- java 21
	- liquidbounce 0.27.1
	   |-- ai_djl_api 0.32.0
	   |-- ai_djl_pytorch_pytorch-engine 0.32.0
	   |-- com_github_ccbluex_discordipc 4.0.0
	   |-- com_github_ccbluex_mc-authlib 1.4.1
	   |-- com_github_ccbluex_netty-httpserver 2.1.1
	   |-- com_kohlschutter_junixsocket_junixsocket-common 2.8.3
	   |-- com_kohlschutter_junixsocket_junixsocket-native-common 2.8.3
	   |-- com_squareup_okhttp3_okhttp 5.0.0-alpha.14
	   |-- com_squareup_okio_okio-jvm 3.9.0
	   |-- com_thealtening_api_api 4.1.0
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- io_netty_netty-codec-socks 4.1.97
	   |-- io_netty_netty-handler-proxy 4.1.97
	   |-- mcef 3.0.5-1.21.4
	   |-- net_fabricmc_tiny-mappings-parser 0.3.0+build.17
	   |-- org_ahocorasick_ahocorasick 0.6.3
	   |-- org_apache_tika_tika-core 2.9.2
	   |-- org_graalvm_js_js-language 24.0.2
	   |-- org_graalvm_polyglot_polyglot 24.0.2
	   |-- org_graalvm_regex_regex 24.0.2
	   |-- org_graalvm_sdk_collections 24.0.2
	   |-- org_graalvm_sdk_jniutils 24.0.2
	   |-- org_graalvm_sdk_nativeimage 24.0.2
	   |-- org_graalvm_sdk_word 24.0.2
	   |-- org_graalvm_shadowed_icu4j 24.0.2
	   |-- org_graalvm_shadowed_json 24.0.2
	   |-- org_graalvm_tools_chromeinspector-tool 24.0.2
	   |-- org_graalvm_tools_coverage-tool 24.0.2
	   |-- org_graalvm_tools_dap-tool 24.0.2
	   |-- org_graalvm_tools_insight-heap-tool 24.0.2
	   |-- org_graalvm_tools_insight-tool 24.0.2
	   |-- org_graalvm_tools_lsp-tool 24.0.2
	   |-- org_graalvm_tools_lsp_api 24.0.2
	   |-- org_graalvm_tools_profiler-tool 24.0.2
	   |-- org_graalvm_truffle_truffle-api 24.0.2
	   |-- org_graalvm_truffle_truffle-compiler 24.0.2
	   \-- org_graalvm_truffle_truffle-runtime 24.0.2
	- minecraft 1.21.4
	- nochatreports 1.21.4-v2.11.0
	- reeses-sodium-options 1.8.3+mc1.21.4
	- sodium 0.6.10+mc1.21.4
	- sodium-extra 0.6.1+mc1.21.4
	- viafabricplus 4.0.3
	   |-- com_github_oryxel_cubeconverter 25bc6ce1b1
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viaaprilfools-common 4.0.1-20250227.200214-7
	   |-- com_viaversion_viabackwards-common 5.2.2-20250225.174312-12
	   |-- com_viaversion_vialoader 4.0.1-20250226.153411-5
	   |-- com_viaversion_viaversion-common 5.2.2-20250227.213313-22
	   |-- de_florianmichael_classic4j 2.1.1-SNAPSHOT
	   |-- io_jsonwebtoken_jjwt-api 0.12.6
	   |-- io_jsonwebtoken_jjwt-gson 0.12.6
	   |-- io_jsonwebtoken_jjwt-impl 0.12.6
	   |-- io_netty_netty-codec-http 4.1.118
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_commons_httpclient 1.6.0
	   |-- net_lenni0451_mcping 1.4.3
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.2
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.2
	   |-- net_lenni0451_reflect 1.4.0
	   |-- net_raphimc_minecraftauth 4.1.1
	   |-- net_raphimc_viabedrock 0.0.15-20250225.225217-3
	   |-- net_raphimc_vialegacy 3.0.8-20250227.101913-11
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR3-SNAPSHOT
	   |-- org_lz4_lz4-pure-java 1.8.0
	   |-- viafabricplus-api 4.0.3
	   |-- viafabricplus-api-legacy 4.0.3
	   \-- viafabricplus-visuals 4.0.3
[20:06:31] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/E:/Games/edr/libraries/net/fabricmc/sponge-mixin/0.15.4+mixin.0.8.7/sponge-mixin-0.15.4+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[20:06:31] [main/INFO]: Compatibility level set to JAVA_21
[20:06:32] [main/INFO]: Loaded configuration file for Sodium: 43 options available, 0 override(s) found
[20:06:32] [main/INFO]: Loaded configuration file for Sodium Extra: 34 options available, 0 override(s) found
[20:06:33] [main/WARN]: Error loading class: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper (java.lang.ClassNotFoundException: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper)
[20:06:34] [main/INFO]: Searching for graphics cards...
[20:06:34] [main/INFO]: Found graphics adapter: AdapterInfo{vendor=INTEL, description='Intel(R) HD Graphics 4600', adapterType=0x00000023, openglIcdFilePath='ig75icd64.dll', openglIcdVersion=20.19.15.4531}
[20:06:34] [main/INFO]: Found graphics adapter: AdapterInfo{vendor=NVIDIA, description='NVIDIA GeForce GTX 750 Ti', adapterType=0x0000030B, openglIcdFilePath='C:\Windows\System32\DriverStore\FileRepository\nv_dispui.inf_amd64_4bc3192b17b6123a\nvoglv64.dll', openglIcdVersion=32.0.15.5585}
[20:06:34] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS_BROKEN, INTEL_FRAMEBUFFER_BLIT_CRASH_WHEN_UNFOCUSED, INTEL_DEPTH_BUFFER_COMPARISON_UNRELIABLE]
[20:06:34] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[20:06:34] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[20:06:39] [Datafixer Bootstrap/INFO]: 243 Datafixer optimizations took 3097 milliseconds
[20:06:48] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[20:06:48] [Render thread/INFO]: Setting user: dawfg
[20:06:49] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[20:06:49] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[20:06:50] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[20:06:50] [Render thread/INFO]: Item Groups: [ Heads, Exploits, Containers ]
[20:06:50] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[20:06:50] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[20:06:52] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:25300/
[20:06:52] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: c0.0.15a-1 (0)
[20:06:52] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 1.21.4 (769)
[20:06:52] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[20:06:53] [Render thread/INFO]: Backend library: LWJGL version 3.3.3-snapshot
[20:06:53] [Render thread/INFO]: Modifying process environment to apply workarounds for the NVIDIA graphics driver...
[20:06:54] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[20:06:54] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 750 Ti/PCIe/SSE2
[20:06:54] [Render thread/INFO]: OpenGL Version: 3.2.0 NVIDIA 555.85
[20:06:54] [Render thread/INFO]: Modifying OpenGL context to apply workarounds for the NVIDIA graphics driver...
[20:06:54] [Render thread/INFO]: Enabling GL_DEBUG_OUTPUT_SYNCHRONOUS to force the NVIDIA driver to disable threadedcommand submission
[20:06:56] [Render thread/INFO]: Launching LiquidBounce v0.27.1 by CCBlueX
[20:06:56] [Render thread/INFO]: Client Version: 0.27.1 (git-f6c59a5)
[20:06:56] [Render thread/INFO]: Client Branch: nextgen
[20:06:56] [Render thread/INFO]: Operating System: Windows 10 (10.0)
[20:06:56] [Render thread/INFO]: Java Version: 21.0.6
[20:06:56] [Render thread/INFO]: Screen Resolution: 854x480
[20:06:56] [Render thread/INFO]: Refresh Rate: 60 Hz
[20:06:58] [Render thread/INFO]: Successfully extracted default theme
[20:06:58] [Render thread/INFO]: Loaded background image for theme default
[20:06:58] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, com_viaversion_viaaprilfools-common, com_viaversion_viabackwards-common, com_viaversion_viaversion-common, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, net_raphimc_viabedrock, net_raphimc_vialegacy, nochatreports, reeses-sodium-options, sodium, sodium-extra, viafabricplus, viafabricplus-visuals
[20:06:59] [Worker-Main-2/INFO]: Found unifont_all_no_pua-16.0.01.hex, loading
[20:07:00] [Worker-Main-2/INFO]: Found unifont_jp_patch-16.0.01.hex, loading
[20:07:04] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers (Realtek(R) Audio)
[20:07:04] [Render thread/INFO]: Sound engine started
[20:07:04] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[20:07:04] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[20:07:04] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[20:07:04] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[20:07:04] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[20:07:04] [Render thread/INFO]: Created: 64x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[20:07:04] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[20:07:04] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[20:07:04] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[20:07:05] [Render thread/INFO]: Probing environment...
[20:07:05] [Render thread/INFO]: Intermediary environment detected.
[20:07:06] [Render thread/INFO]: [ScriptAPI] Engine Version: Development Build, Supported languages: [ js ]
[20:07:06] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor.
[20:07:06] [Render thread/INFO]: Loaded language en_us
[20:07:07] [DefaultDispatcher-worker-9/INFO]: Successfully loaded Discord IPC configuration [443472046031110144].
[20:07:09] [DefaultDispatcher-worker-8/INFO]: Successfully loaded 956 heads from HeadDB
[20:07:09] [Render thread/INFO]: Default port 15000 available.
[20:07:09] [Render thread/INFO]: JCEF Commit: 06399a49d24f2c372668ccce26a15c533199ec9e
[20:07:09] [netty-websocket/INFO]: Starting Netty server...
[20:07:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x939b3233] REGISTERED
[20:07:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x939b3233] BIND: 0.0.0.0/0.0.0.0:15000
[20:07:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x939b3233] CLOSE
[20:07:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x939b3233] UNREGISTERED
[20:07:09] [netty-websocket/ERROR]: Netty server failed - 15000
java.net.BindException: Address already in use: bind
	at java.base/sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.bind(Unknown Source) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(Unknown Source) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(Unknown Source) ~[?:?]
	at knot/io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:141) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:561) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1281) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230) ~[netty-handler-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:602) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:922) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel.bind(AbstractChannel.java:259) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:380) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.115.Final.jar:?]
	at java.base/java.lang.Thread.run(Unknown Source) ~[?:?]
[20:07:09] [netty-websocket/ERROR]: Fatal error
java.net.BindException: Address already in use: bind
	at java.base/sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.bind(Unknown Source) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(Unknown Source) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(Unknown Source) ~[?:?]
	at knot/io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:141) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:561) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1281) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230) ~[netty-handler-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:602) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:922) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel.bind(AbstractChannel.java:259) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:380) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.115.Final.jar:?]
	at java.base/java.lang.Thread.run(Unknown Source) ~[?:?]
[20:07:09] [DefaultDispatcher-worker-1/INFO]: Downloading checksum file... [1/3]
[20:07:09] [DefaultDispatcher-worker-8/INFO]: [DeepLearning] Initializing engine...
[20:07:10] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/zlibwapi.dll.gz ...
[20:07:10] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cupti64_2024.1.0.dll.gz ...
[20:07:10] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_ops64_9.dll.gz ...
[20:07:10] [Render thread/WARN]: Failed to place all characters (4104) on the atlas, using a reduced charset (3600) instead!
[20:07:10] [DefaultDispatcher-worker-1/INFO]: Downloading JCEF... [1/3]
[20:07:10] [Render thread/INFO]: Completed loading fonts in 1182 ms.
[20:07:11] [Render thread/INFO]: Fonts: [ Segoe UI, Inter Bold, Inter Light, Inter Medium, Inter Regular ]
[20:07:11] [Render thread/INFO]: Inserted 15 components
[20:07:11] [Render thread/INFO]: Successfully loaded config 'theme'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'theme'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'modules'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'modules'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'commands'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'commands'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'proxy'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'proxy'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'accounts'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'accounts'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'friends'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'friends'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'tabs'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'tabs'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'lang'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'lang'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'account'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'account'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'spoofer'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'spoofer'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'combattargets'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'combattargets'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'visualtargets'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'visualtargets'.
[20:07:11] [Render thread/INFO]: Successfully loaded config 'storage'.
[20:07:11] [Render thread/INFO]: Successfully saved config 'storage'.
[20:07:11] [Render thread/INFO]: LiquidBounce has been successfully initialized.
[20:07:11] [Render thread/INFO]: Successfully loaded 432 cosmetics carriers.
[20:07:11] [DefaultDispatcher-worker-9/ERROR]: Failed to connect to Discord RPC.
com.jagrosh.discordipc.exceptions.NoDiscordClientException: No Valid Discord Client was found for this Instance
	at knot/com.jagrosh.discordipc.entities.pipe.Pipe.openPipe(Pipe.java:163) ~[com_github_ccbluex_discordipc-4.0.0-e5ca8b6597c4be2b.jar:?]
	at knot/com.jagrosh.discordipc.IPCClient.connect(IPCClient.java:375) ~[com_github_ccbluex_discordipc-4.0.0-e5ca8b6597c4be2b.jar:?]
	at knot/net.ccbluex.liquidbounce.features.module.modules.client.ModuleRichPresence.connectIpc(ModuleRichPresence.kt:94) [LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.features.module.modules.client.ModuleRichPresence.access$connectIpc(ModuleRichPresence.kt:60) [LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.features.module.modules.client.ModuleRichPresence$updateCycle$1$1.invokeSuspend(ModuleRichPresence.kt:143) [LiquidBounce.jar:?]
	at knot/kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [org_jetbrains_kotlin_kotlin-stdlib-2.1.10-f121dd60ab1b676b.jar:?]
	at knot/kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
[20:07:17] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_cnn64_9.dll.gz ...
[20:07:19] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/torch.dll.gz ...
[20:07:19] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cufftw64_11.dll.gz ...
[20:07:19] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/nvrtc64_120_0.dll.gz ...
[20:07:20] [DefaultDispatcher-worker-1/INFO]: Extracting JCEF...
[20:07:20] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cusolverMg64_11.dll.gz ...
[20:07:22] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/torch_cuda.dll.gz ...
[20:07:23] [Render thread/INFO]: Initializing browser...
[20:07:23] [Render thread/INFO]: Initializing CEF on windows_amd64...
[20:07:23] [DefaultDispatcher-worker-5/INFO]: Requesting to login into Mojang...
[20:07:24] [nioEventLoopGroup-5-1/WARN]: Failed authentication to LiquidChat
[20:07:28] [Render thread/INFO]: [STDOUT]: initialize on Thread[#1,Render thread,5,main] with library path E:\Games\edr\gameDir\nextgen\LiquidBounce\mcef\libraries\06399a49d24f2c372668ccce26a15c533199ec9e\windows_amd64
[20:07:28] [Render thread/INFO]: Chromium Embedded Framework initialized
[20:07:28] [Render thread/INFO]: Browser is ready.
[20:07:28] [Render thread/INFO]: Successfully initialized browser.
[20:07:55] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cufft64_11.dll.gz ...
[20:08:09] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cublas64_12.dll.gz ...
[20:08:12] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_graph64_9.dll.gz ...
[20:08:12] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/uv.dll.gz ...
[20:08:12] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/caffe2_nvrtc.dll.gz ...
[20:08:12] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/torch_cpu.dll.gz ...
[20:08:15] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/c10.dll.gz ...
[20:08:16] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_adv64_9.dll.gz ...
[20:08:24] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/c10_cuda.dll.gz ...
[20:08:24] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudart64_12.dll.gz ...
[20:08:24] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/fbgemm.dll.gz ...
[20:08:24] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/curand64_10.dll.gz ...
[20:08:27] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_engines_runtime_compiled64_9.dll.gz ...
[20:08:27] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/nvrtc-builtins64_124.dll.gz ...
[20:08:27] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_heuristic64_9.dll.gz ...
[20:08:29] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/libiomp5md.dll.gz ...
[20:08:29] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cusolver64_11.dll.gz ...
[20:08:34] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn64_9.dll.gz ...
[20:08:34] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cublasLt64_12.dll.gz ...
[20:08:53] [DefaultDispatcher-worker-8/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cudnn_engines_precompiled64_9.dll.gz ...
[20:09:17] [DefaultDispatcher-worker-3/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/nvToolsExt64_1.dll.gz ...
[20:09:17] [DefaultDispatcher-worker-3/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/nvJitLink_120_0.dll.gz ...
[20:09:18] [DefaultDispatcher-worker-3/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/cusparse64_12.dll.gz ...
[20:09:30] [DefaultDispatcher-worker-3/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/asmjit.dll.gz ...
[20:09:30] [DefaultDispatcher-worker-3/INFO]: Downloading https://publish.djl.ai/pytorch/2.5.1/cu124/win-x86_64/native/lib/libiompstubs5md.dll.gz ...
[20:09:36] [DefaultDispatcher-worker-3/INFO]: Failed to initialize deep learning.
ai.djl.engine.EngineException: Failed to load PyTorch native library
	at knot/ai.djl.pytorch.engine.PtEngine.newInstance(PtEngine.java:90) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	at knot/ai.djl.pytorch.engine.PtEngineProvider.getEngine(PtEngineProvider.java:41) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	at knot/ai.djl.engine.Engine.getEngine(Engine.java:190) ~[ai_djl_api-0.32.0-baa881eac3391b62.jar:?]
	at knot/ai.djl.engine.Engine.getInstance(Engine.java:145) ~[ai_djl_api-0.32.0-baa881eac3391b62.jar:?]
	at knot/net.ccbluex.liquidbounce.deeplearn.DeepLearningEngine$init$engine$1.invokeSuspend(DeepLearningEngine.kt:78) ~[LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.deeplearn.DeepLearningEngine$init$engine$1.invoke(DeepLearningEngine.kt) ~[LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.deeplearn.DeepLearningEngine$init$engine$1.invoke(DeepLearningEngine.kt) ~[LiquidBounce.jar:?]
	at knot/kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:43) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.BuildersKt.withContext(Unknown Source) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/net.ccbluex.liquidbounce.deeplearn.DeepLearningEngine.init(DeepLearningEngine.kt:77) ~[LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce$prepareGuiStage$1$1.invokeSuspend(LiquidBounce.kt:309) [LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce$prepareGuiStage$1$1.invoke(LiquidBounce.kt) [LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce$prepareGuiStage$1$1.invoke(LiquidBounce.kt) [LiquidBounce.jar:?]
	at knot/net.ccbluex.liquidbounce.integration.task.TaskManager$launch$1.invokeSuspend(TaskManager.kt:70) [LiquidBounce.jar:?]
	at knot/kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [org_jetbrains_kotlin_kotlin-stdlib-2.1.10-f121dd60ab1b676b.jar:?]
	at knot/kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
	at knot/kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704) [org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.1-e60f893de62e97b9.jar:?]
Caused by: java.lang.UnsatisfiedLinkError: E:\Games\edr\gameDir\nextgen\LiquidBounce\deeplearning\engines\pytorch\2.5.1-cu124-win-x86_64\torch_cuda.dll: Can't find dependent libraries
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method) ~[?:?]
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(Unknown Source) ~[?:?]
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(Unknown Source) ~[?:?]
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(Unknown Source) ~[?:?]
	at java.base/java.lang.ClassLoader.loadLibrary(Unknown Source) ~[?:?]
	at java.base/java.lang.Runtime.load0(Unknown Source) ~[?:?]
	at java.base/java.lang.System.load(Unknown Source) ~[?:?]
	at knot/ai.djl.pytorch.jni.LibUtils.loadNativeLibrary(LibUtils.java:379) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	at knot/ai.djl.pytorch.jni.LibUtils.loadLibTorch(LibUtils.java:195) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	at knot/ai.djl.pytorch.jni.LibUtils.loadLibrary(LibUtils.java:82) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	at knot/ai.djl.pytorch.engine.PtEngine.newInstance(PtEngine.java:53) ~[ai_djl_pytorch_pytorch-engine-0.32.0-6587810b3cadf766.jar:?]
	... 22 more
[20:10:12] [Thread-24/INFO]: Shutting down client...
```

### Screenshots

![Image](https://github.com/user-attachments/assets/7f123092-1210-4fa0-9904-b86dcd5e8bd8)

--- a/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
@@ -30,8 +30,8 @@ import net.ccbluex.netty.http.HttpServer
 import net.ccbluex.netty.http.middleware.CorsMiddleware
 import net.ccbluex.netty.http.model.RequestObject
 import net.ccbluex.netty.http.util.httpOk
+import java.net.BindException
 import java.net.Socket
-import kotlin.concurrent.thread
 
 /**
  * A client server implementation.
@@ -50,7 +50,7 @@ object ClientInteropServer {
             logger.info("Default port unavailable. Falling back to random port.")
             (15001..17000).random()
         }
-    } catch (expected: Exception) {
+    } catch (_: Exception) {
         logger.info("Default port $DEFAULT_PORT available.")
 
         DEFAULT_PORT
@@ -74,10 +74,29 @@ object ClientInteropServer {
         }.onFailure(ErrorHandler::fatal)
 
         // Start the HTTP server
-        thread(name = "netty-websocket") {
-            runCatching {
-                httpServer.start(port)
-            }.onFailure(ErrorHandler::fatal)
+        startServer()
+    }
+
+    private var attempt = 0
+
+    private fun startServer(port: Int = this.port) {
+        runCatching {
+            httpServer.start(port)
+        }.onFailure { error ->
+            when (error) {
+                is BindException -> {
+                    if (attempt >= 5) {
+                        ErrorHandler.fatal(error)
+                        return@onFailure
+                    }
+
+                    // Retry with random port
+                    attempt++
+                    logger.error("Failed to bind to port $port. Falling back to random port.")
+                    startServer((15001..17000).random())
+                }
+                else -> ErrorHandler.fatal(error)
+            }
         }
     }
 

--- a/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
@@ -78,25 +78,21 @@ object ClientInteropServer {
     }
 
     private var attempt = 0
-
     private fun startServer(port: Int = this.port) {
-        runCatching {
+        try {
             httpServer.start(port)
-        }.onFailure { error ->
-            when (error) {
-                is BindException -> {
-                    if (attempt >= 5) {
-                        ErrorHandler.fatal(error)
-                        return@onFailure
-                    }
-
-                    // Retry with random port
-                    attempt++
-                    logger.error("Failed to bind to port $port. Falling back to random port.")
-                    startServer((15001..17000).random())
-                }
-                else -> ErrorHandler.fatal(error)
+        } catch (bindException: BindException) {
+            if (attempt >= 5) {
+                ErrorHandler.fatal(bindException)
+                return
             }
+
+            // Retry with random port
+            attempt++
+            logger.error("Failed to bind to port $port. Falling back to random port.")
+            startServer((15001..17000).random())
+        } catch (exception: Exception) {
+            ErrorHandler.fatal(exception)
         }
     }
 

